pvactools/lib/load_gtf_data.py

```diff
@@ -0,0 +1,42 @@
+import gtfparse
+import sys
+
+
+class LoadGtfData:
+    def __init__(self, **kwargs):
+        self.gtf_file    = kwargs['gtf_file']
+        self.output_file = kwargs['output_file']
+        self.save_gtf    = kwargs['save_gtf']  # default false
+        self.tsl         = kwargs['tsl']
+        if not isinstance(self.tsl, int):
+            raise ValueError('Input TSL value should be a positive integer from 1-5!')
+
+    def execute(self):
+
+        # make sure running pandas not polars
+        gtf_df_all = gtfparse.read_gtf(self.gtf_file, usecols=[
+            'feature', 'seqname', 'start', 'end', 'transcript_id',
+            'transcript_biotype', 'transcript_version', 'transcript_support_level', 'exon_number', 'gene_name',
+            'gene_id'], result_type='pandas')
+
+        gtf_df_all = gtf_df_all[~gtf_df_all['transcript_support_level'].isin(['', 'NA', 'Not Supported'])]
```

Comment:
I'm not sure that this is what we want. E.g., if someone runs GRCh37, there is no TSL data available and we would write 'Not Supported' to all entries. But then all transcripts would be included with this filtering step. I think 'Not Supported' is something we explicitly want to include.

Reply:
I went back and looked at some examples of Ensembl gtf files and had a couple additional thoughts:
- For GRCh37, the gtf file ([example](http://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/)) doesn't contain a "transcript_support_level" key. The `Not Supported` terminology is something we came up with for handling GRCh37 in pVACseq, not something straight from Ensembl. In pVACseq we check the build version of the VCF (see https://github.com/griffithlab/pVACtools/blob/staging/pvactools/lib/input_file_converter.py#L129) before parsing the TSL field. GTF files have a genome-version header that we could use for this purpose. If it's not GRCh38 we shouldn't do any of the TSL filtering.
- Looking at [GRCh38](http://ftp.ensembl.org/pub/release-106/gtf/homo_sapiens/), I see some transcript support level values like this: `"1 (assigned to previous version 3)"`. Does the gtfparse module already handle this or do we need to do some special parsing here? Or does the `astype` below end up converting this correctly?

Reply:
I do some special parsing to make sure there's no string in the final TSL col

Reply:
I'll double check about the Not Supported